### PR TITLE
testing/emulationstation: new aport

### PR DESCRIPTION
**Pull request by tanertas**
https://github.com/RetroPie/EmulationStation
EmulationStation is a cross-platform graphical front-end for emulators with controller navigation.

### CODE REVIEW

File: testing/emulationstation/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+pkgver=2.7.1
+pkgrel=0
+pkgdesc="EmulationStation is a cross-platform graphical front-end for emulators with controller navigation."
+url="https://github.com/RetroPie/EmulationStation"
+arch="all"
+license="MIT"
+depends="$pkgname-theme-default"
+makedepends="boost-dev freeimage-dev freetype-dev sdl2-dev mesa-dev alsa-lib-dev vlc-dev curl-dev cmake"
+subpackages="$pkgname-theme-default:theme:noarch"
+options="!check"
```
Comment (TBK) [inline testing/emulationstation/APKBUILD]: Please add a comment why check is disabled.

File: testing/emulationstation/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+_distname="EmulationStation"
```
Comment (TBK) [inline testing/emulationstation/APKBUILD]: This is normally done with `_pkgname` placed just after `pkgname`.

File: testing/emulationstation/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+_pugixml_git="https://github.com/zeux/pugixml.git"
```
Comment (TBK) [inline testing/emulationstation/APKBUILD]: pugixml should be its own aport or atleast pulled a taged release through the `source=""` mechanisme to make the build reproducible.

File: testing/emulationstation/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/RetroPie/${_distname}/archive/v${pkgver}.tar.gz
```
Comment (TBK) [inline testing/emulationstation/APKBUILD]: Wrapping the variables in {} is not needed in most cases.

File: testing/emulationstation/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+	mkdir -p "$pkgdir"/usr/share/${pkgname}/gamelists
+	mkdir -p "$pkgdir"/usr/share/${pkgname}/themes
+	install -Dm755 emulationstation "$pkgdir"/usr/bin/emulationstation
+	install -Dm644 "$srcdir"/${pkgname}.png "$pkgdir"/usr/share/pixmaps/
+	install -Dm644 "$srcdir"/${pkgname}.desktop "$pkgdir"/usr/share/applications/
+}
+
+theme() {
+	local _theme_git="https://github.com/rxbrad/es-theme-gbz35.git"
+	local _theme_name="gbz35"
```
Comment (TBK) [inline testing/emulationstation/APKBUILD]: Should be pulled through `source=""`, would be great if you could get upstream to make a tagged release and use that instead.

File: testing/emulationstation/APKBUILD (hunk 1 above)
Comment (tanertas) [reply testing/emulationstation/APKBUILD]: Done.

File: testing/emulationstation/APKBUILD (hunk 2 above)
Comment (tanertas) [reply testing/emulationstation/APKBUILD]: Done.

File: testing/emulationstation/APKBUILD (hunk 3 above)
Comment (tanertas) [reply testing/emulationstation/APKBUILD]: Done.

File: testing/emulationstation/APKBUILD (hunk 4 above)
Comment (tanertas) [reply testing/emulationstation/APKBUILD]: Done.

File: testing/emulationstation/APKBUILD (hunk 5 above)
Comment (tanertas) [reply testing/emulationstation/APKBUILD]: Done.

**Comment by ncopa** [comment]
can you please merge the commits into the first? Keep commit message in first commit.

**Comment by tanertas** [comment]
Done.

File: testing/emulationstation/APKBUILD [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+	emulationstation.desktop
+	emulationstation.png"
+builddir="$srcdir"/$_pkgname-$pkgver
+
+prepare() {
+	cd "$builddir"
+        if [ ! -d external/pugixml.dist ]; then
+        mv -f external/pugixml external/pugixml.dist
+	cp -aP "$srcdir"/pugixml-$_pugixml_commit external/pugixml
+        fi
```
Comment (ncopa) [inline testing/emulationstation/APKBUILD]: looks like whitespace damage above.

**Pull request closed without merging**